src/proposals/20251211/SparkAvalanche_20251211.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.25;
+
+import { ForeignController } from "spark-alm-controller/src/ForeignController.sol";
+
+import { SparkPayloadAvalanche, Avalanche } from "../../SparkPayloadAvalanche.sol";
+
+import { ISparkVaultV2Like } from "../../interfaces/Interfaces.sol";
+
+/**
+ * @title  December 11, 2025 Spark Avalanche Proposal
+ * @notice Spark Savings - Update Setter Role to ALM Proxy Freezable for spUSDC
+ * Forum:  https://forum.sky.money/t/december-11-2025-proposed-changes-to-spark-for-upcoming-spell/27481
+ * Vote:   
+ */
+contract SparkAvalanche_20251211 is SparkPayloadAvalanche {
+
+    address internal constant ALM_PROXY_FREEZABLE = 0x45d91340B3B7B96985A72b5c678F7D9e8D664b62;
+
+    function execute() external {
+        // Grant CONTROLLER Role for Relayer 1 and 2 on ALM_PROXY_FREEZABLE and Freezer role to the ALM_FREEZER_MULTISIG
+        IALMProxy(ALM_PROXY_FREEZABLE).grantRole(
+            IALMProxy(ALM_PROXY_FREEZABLE).CONTROLLER(),
+            Avalanche.ALM_RELAYER
+        );
+        IALMProxy(ALM_PROXY_FREEZABLE).grantRole(
+            IALMProxy(ALM_PROXY_FREEZABLE).CONTROLLER(),
+            Avalanche.ALM_RELAYER2
+        );
+        IALMProxy(ALM_PROXY_FREEZABLE).grantRole(
+            IALMProxyFreezableLike(ALM_PROXY_FREEZABLE).FREEZER(),
+            Avalanche.ALM_FREEZER
+        );
+
+        // Spark Savings - Update Setter Role to ALM Proxy Freezable for spUSDC
+        ISparkVaultV2Like(Avalanche.SPARK_VAULT_V2_SPUSDC).grantRole(ISparkVaultV2Like(Avalanche.SPARK_VAULT_V2_SPUSDC).SETTER_ROLE(), ALM_PROXY_FREEZABLE);
```

Comment:
Need to revoke the setter role from the ALM_OPS_MULTISIG for all of these operations on all chains

Reply:
done